ssl/test/bssl_shim.cc

```diff
@@ -905,6 +1038,30 @@ static bool DoExchange(bssl::UniquePtr<SSL_SESSION> *out_session,
       });
     } while (RetryAsync(ssl, ret));
 
+    CheckSSLTransfer(config, ssl);
+
+    // Fetch tls_unique_len_before_transfer before SSL transfer.
```

Comment:
What is this?

Reply:
This is to fetch the value before ssl transfer, and then use this value to compare the state after transfer.

Reply:
I mean, what value is it in relation to TLS? Not sure what `tls-unique` is.

Is it the server/client random?

Reply:
> I mean, what value is it in relation to TLS? Not sure what tls-unique is.

https://datatracker.ietf.org/doc/html/rfc5929#section-3.1

https://github.com/awslabs/aws-lc/blob/c30170a471a18455b8eb54002605859244d906e9/include/openssl/ssl.h#L1617-L1635